cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -216,3 +210,24 @@ class InformationRules(RuleModel):
     metadata: InformationMetadata = Field(alias="Metadata")
     properties: SheetList[InformationProperty] = Field(alias="Properties")
     classes: SheetList[InformationClass] = Field(alias="Classes")
+
+    @model_validator(mode="after")
+    def update_entities_prefix(self) -> Self:
+        prefix = self.metadata.prefix
```

Comment:
Maybe more readable?
```suggestion
        default_prefix = self.metadata.prefix
```